spec/models/outcome_coverage_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe OutcomeCoverage do
+  context "validations" do
+    subject do
```

Comment:
I have a personal preference to do this inside an "it" block rather than in the "subject" -- particularly since the subject really only applies to this one test case. You can see how I did this here: https://github.com/MIT-IR/abet/pull/201/files#diff-d05b36864e0c1f4fc61a91c251004dbeR4

I don't feel strongly about this though. If you prefer subject then I think that's fine here.